exchanges/poloniex/poloniex_websocket.go

```diff
@@ -28,1126 +27,561 @@ import (
 )
 
 const (
-	poloniexWebsocketAddress = "wss://api2.poloniex.com"
-	wsAccountNotificationID  = 1000
-	wsTickerDataID           = 1002
-	ws24HourExchangeVolumeID = 1003
-	wsHeartbeat              = 1010
+	poloniexWebsocketAddress        = "wss://ws.poloniex.com/ws/public"
+	poloniexPrivateWebsocketAddress = "wss://ws.poloniex.com/ws/private"
```

Comment:
🚧 Change request:
* Everything is poloniex in this package.
* URL vs Address
```suggestion
	websocketURL        = "wss://ws.poloniex.com/ws/public"
	privateWebsocketURL = "wss://ws.poloniex.com/ws/private"
```